tests/x11/gnucash.pm

```diff
@@ -27,7 +27,8 @@ sub run {
     # Leave tutorial window
     wait_screen_change { send_key 'alt-f4' };
     # Leave tips windows for GNOME case
-    if (check_var('DESKTOP', 'gnome') || check_var('DESKTOP', 'xfce')) {
+    if (check_var('DESKTOP', 'gnome') || check_var('DESKTOP', 'xfce') || check_var('DESKTOP', 'lxde')) {
```

Comment:
I guess this would look nicer by now with `if (get_var('DESKTOP', '') =~ /gnome|xfce/lxde/)` but leave that for next time :)

Reply:
Thats some really pretty code, I hope I can remember this next time I need it